### DISCUSSION TITLE
Preset & Component request: Evergreen UI

### DISCUSSION THREAD

**Discussion by a-y-u-s-h** [system-ui/theme-ui#1196] [Ideas]
**Is your feature request related to a problem? Please describe.**
I love Theme UI and the concept of presets. It contains some awesome presets but it doesn't have a preset for [Evergreen UI](https://evergreen.segment.com). 

**Describe the solution you'd like**
 I would like to request a preset that comes as close as possible to [Evergreen UI](https://evergreen.segment.com). There are  some great components in Evergreen UI as well which Theme UI doesn't have yet. (Corner Dialog, SIde Sheets, Table, etc) so I'd like to suggest this enhancement as well. 

**Describe alternatives you've considered**
Evergreen UI itself.

**Comment by atanasster** [comment]
Hi @a-y-u-s-h - that would be great. Can you make it happen, or we could work together on a PR?

**Comment by a-y-u-s-h** [comment]
Nature of my work keeps me from working too much on Theme UI but I'm slowly making in this as well (basically copying the styles from devtools and making a ThemeUI preset) but I think you should start as well you'll probably finish faster working independently as I'm under other kind of work pressure. Theme UI lacks some components like PopOver and Menu (utilitiy components as well, like Portal) necessary to make some of the components of Evergreen UI. 

I also came across this wonderful library that has components styled to give a native desktop feel of MacOS or a Vanilla native system - [Xel](https://xel-toolkit.org/). Going to spend some time on weekends turning this into a preset as well.

**Comment by atanasster** [comment]
Very cool, keep us updated
